config/config.exs

```diff
@@ -5,5 +5,7 @@ config :rclex,
     "std_msgs/msg/String",
     "std_msgs/msg/UInt32MultiArray",
     "geometry_msgs/msg/Twist",
-    "sensor_msgs/msg/PointCloud"
+    "sensor_msgs/msg/PointCloud",
+    "diagnostic_msgs/msg/KeyValue",
+    "diagnostic_msgs/msg/DiagnosticStatus"
```

Comment:
```suggestion
    "sensor_msgs/msg/PointCloud",
    "diagnostic_msgs/msg/DiagnosticStatus"
```

We don't need the deps of `DiagnosticStatus`, `KeyValue` in config. `mix rclex.gen.msgs` find it and generate its code.